.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on:
+  - pull_request
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Bun
+        uses: oven-sh/setup-bun@v2
+
+      - name: Install dependencies
+        run: bun install
```

Comment:
@Rindrics It is better to use the bun version in package.json.

https://github.com/route06inc/giselle/blob/930deb6b9306a2627c0907005c4ed58b1cb52f88/.github/workflows/license.yml#L27-L30

https://github.com/route06inc/giselle/blob/930deb6b9306a2627c0907005c4ed58b1cb52f88/package.json#L14

Reply:
I checked that it has been replaced by https://github.com/route06inc/giselle/pull/47